other/upgrade_2-1_postgresql.sql

```diff
@@ -2681,6 +2708,12 @@ if (!array_key_exists($modSettings['smiley_sets_default'], $filtered))
 ---}
 ---#
 
+---# Drop obsolete filename column from smileys table
+ALTER TABLE {$db_prefix}smileys
+DROP COLUMN IF EXISTS filename;
```

Comment:
I'm nervous about dropping the filename column unconditionally here.  Any hiccup above, and this is executed, then the filenames are lost.  I would suggest doing this via db_query within the code block above.